rpcserver.go

```diff
@@ -2570,80 +2570,92 @@ func (r *rpcServer) CommitVirtualPsbts(ctx context.Context,
 		return nil, fmt.Errorf("error serializing packet: %w", err)
 	}
 
-	// The change output and fee parameters of this RPC are identical to the
-	// walletrpc.FundPsbt, so we just map them 1:1 and let lnd do the
-	// validation.
-	coinSelect := &walletrpc.PsbtCoinSelect{
-		Psbt: packetBytes,
-	}
-	fundRequest := &walletrpc.FundPsbtRequest{
-		Template: &walletrpc.FundPsbtRequest_CoinSelect{
-			CoinSelect: coinSelect,
-		},
-		MinConfs:              1,
-		ChangeType:            P2TRChangeType,
-		CustomLockId:          req.CustomLockId,
-		LockExpirationSeconds: req.LockExpirationSeconds,
-	}
+	var (
+		lockedUTXO      []*walletrpc.UtxoLease
+		lockedOutpoints []wire.OutPoint
+	)
 
-	// Unfortunately we can't use the same RPC types, so we have to do a
-	// 1:1 mapping to the walletrpc types for the anchor change output and
-	// fee "oneof" fields.
-	type existingIndex = walletrpc.PsbtCoinSelect_ExistingOutputIndex
-	switch change := req.AnchorChangeOutput.(type) {
-	case *wrpc.CommitVirtualPsbtsRequest_ExistingOutputIndex:
-		coinSelect.ChangeOutput = &existingIndex{
-			ExistingOutputIndex: change.ExistingOutputIndex,
-		}
+	fundedPacket := pkt
+	changeIndex := int32(-1)
+	success := false
```

Comment:
nit: merge this block into the `var` statement above?

Reply:
done